integration/rocket-chat/main.js

```diff
@@ -525,8 +525,17 @@ function SmartiWidget(element,_options) {
             resultCount.empty();
             resultPaging.empty();
             loader.show();
+
+            let customSuffix = RocketChat.settings.get('Assistify_AI_DBSearch_Suffix') || '';
```

Comment:
You should not access RocketChat.Settings at this point as the scripts is able to run outside RocketChat (so the object would be undefined). It would be better to do this on SmartiWidget initialization with options. (`new SmartiWidget(element,_options`). The options could look like:
```
var options = {
        ...
        widget:{
            'query.dbsearch': {
                numOfResults:2,
                suffix: '&fq=a:b'
            }
        },
        lang:'de'
    };
```